brianlong/www.validators.app
# 184911359 - fix nodes counts

#### What's this PR do?
PR fixes both total and delinquent values for stake pool nodes section.

#### How should this be manually tested?
- locally please  run `rails r script/gather_stake_accounts.rb` and check if no errors presented
- verify values in the "Stake Pools Overview" and each of the stake pool Statistics page, focussing on the Nodes section

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184911359)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
